Documentation/PageTsconfig/Templates.rst

```diff
@@ -114,20 +114,20 @@ into smaller partial files so an extension can override a specific partial only.
 Combinations of overrides
 =========================
 
-Due to the nature of TsConfig and its two types PageTsConfig and UserTsConfig,
+Due to the nature of TSconfig and its two types page TSconfig and user TSconfig,
 various combinations are possible:
 
-* Define "global" overrides with PageTsConfig in
+* Define "global" overrides with page TSconfig in
   :file:`Configuration/page.tsconfig` of an extension. This works for all
   modules, regardless of whether the module renders a page tree or not.
 
-* Define page-level overrides via the :guilabel:`TSconfig` field of page
-  records. As always with PageTsConfig, sub pages and sub trees inherit these
+* Define page level overrides via the :guilabel:`TSconfig` field of page
```

Comment:
I'd keep this "-" because it's "page-level" attributed, has a different relation if the "-" is missing?!

Reply:
I checked with deepl the spelling, it hasn't suggested me the dashed version.

Reply:
Let's make a LLM cage-fight then:

<img width="838" alt="Screenshot 2024-01-22 at 12 51 54" src="https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TSconfig/assets/273326/5bff81a9-2965-45bf-ade9-284190904e83">

*g*

Reply:
@sarahmccarthy123 what is your view as a native speaker on this?

Reply:
@brotkrueml I don't want this "-"-fight to prevent the PR from getting merge I yield to whatever you decide :-P